recipes/torchcam/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "torchcam" %}
+{% set version = "0.3.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/torchcam-{{ version }}.tar.gz
+  sha256: 400d451cbbafe0dbddbf3a22feab9e23d1317e90cf6eae16bca96e0e2ff4f076
+
+build:
+  number: 0
+  noarch: python
+  script:     
+    ## Ensure exclusion of tests folder
+    {% set testsfolder = "test" %}
+    - echo -e "\n>>> ENSURE EXCLUSION OF FOLDER '{{ testsfolder }}' <<<\n\n"
+    - rm -rf {{ testsfolder }}  # [not win]
+    - rmdir /s /q "{{ testsfolder | replace('/', '\\') }}"  # [win]
+    ## Install package {{ name }} from PyPI
```

Comment:
You can reduce these 6 lines to just a `rm -rf test`. No need for the comments b/c the commands are quite descriptive and no need to do that on Windows b/c the recipe is noarch.

Reply:
Done.

Reply:
The comments are not really needed and you can remove the selector too. Also, the jinja won't work on a comment. Merging this as these are all minor comments.